core/dive.c

```diff
@@ -3688,6 +3688,8 @@ int get_surface_pressure_in_mbar(const struct dive *dive, bool non_null)
 	return mbar;
 }
 
+#define SALINITY_TO_SPECIFIC_WEIGHT(s) ((s) * 0.981 / 100000.0)
```

Comment:
ok, now I'm confused. Why are you dividing by 100 thousand?
What are the expected units in and out for this macro?

Reply:
This takes a salinity in the kg/10l unit that we use for salinity and returns a number in mbar/mm that can directly be used to convert between our beloved units of mbar and mm. To me this looks like the only sane way (besides changing the unit of salinity by a factor of 10 everywhere).

Reply:
err, but mbar are not mm. See below. mbar are cm 
so you are no doing deci-mili bar?

Reply:
so we had mbar and cm and that mostly made sense to me. but 1/10mbar and mm doesn't, really.

Reply:
No no no. The task at hand is: Convert a depth (which is usually expressed mm) to a pressure (usually expresses in mm). 
For this, we need a salinity. This macro takes a salinity (in our weird units where sea water has a salinity of 10300) and spits out the conversion factor that does the actual job above without further ad hoc factors.

so, if you like it is 0.981 / 100000 (mbar / mm) / (g / 10l) as our units are mbar for pressure, mbar for depth and g/10l for salinity. 

I think this is natural given our choice of units.

Reply:
To avoid confusion, I would suggest to put the units of measure of the input and output values in a comment...?